draft-ietf-httpbis-cache-latest.xml

```diff
@@ -973,6 +973,29 @@
    the client is referring specifically to one or more previously obtained
    representations with the listed entity-tags).
 </t>
+<t>
+   A cache:
+</t>
+<ul>
+   <li>&MUST; send the relevant entity-tags in any cache validation request
+     (using <x:ref>If-Match</x:ref>, <x:ref>If-None-Match</x:ref>, or
+     <x:ref>If-Range</x:ref>) if the entity-tags were provided in the
+     stored response(s) being validated.</li>
+   <li>&SHOULD; send the <x:ref>Last-Modified</x:ref> value in non-subrange
+     cache validation requests (using <x:ref>If-Modified-Since</x:ref>) if
+     a single stored response is being validated and that response contains
+     a Last-Modified value.</li>
+   <li>&MAY; send the <x:ref>Last-Modified</x:ref> value in subrange cache
+     validation requests (using <x:ref>If-Unmodified-Since</x:ref> or
+     <x:ref>If-Range</x:ref>) if a single stored response is being validated
+     and that response contains only a Last-Modified value
+     (not an entity-tag).</li>
+</ul>
```

Comment:
```suggestion
<t>
   When generating a conditional request for validation, a cache:
</t>
<ul>
   <li>&MUST; send the relevant entity-tags
     (using <x:ref>If-Match</x:ref>, <x:ref>If-None-Match</x:ref>, or
     <x:ref>If-Range</x:ref>) if the entity-tags were provided in the
     stored response(s) being validated.</li>
   <li>&SHOULD; send the <x:ref>Last-Modified</x:ref> value (using 
     <x:ref>If-Modified-Since</x:ref>) if the request is not for a subrange, 
     a single stored response is being validated, and that response 
     contains a Last-Modified value.</li>
   <li>&MAY; send the <x:ref>Last-Modified</x:ref> value (using 
     <x:ref>If-Unmodified-Since</x:ref> or <x:ref>If-Range</x:ref>) if 
     the request is for a subrange, a single stored response is being 
     validated, and that response contains only a Last-Modified value
     (not an entity-tag).</li>
</ul>
```

Just specifying 'A cache... must send...' doesn't match the original text; it's when the cache chooses to generate a conditional request that's important.